src/content/docs/en/guides/routing.mdx

```diff
@@ -296,6 +296,60 @@ if (!isLoggedIn(cookie)) {
 </html>
 ```
 
+## Rewrites
+
+<p><Since v="4.13.0" /></p>
+
+A rewrite allows you serve a different route without redirecting the browser to a different page. The browser will show the original address in the URL bar, but will instead display the content of the URL provided to [`Astro.rewrite()`](/en/reference/api-reference/#astrorewrite).
+
+:::tip
+For content that has permanently moved, or to direct your user to a different page with a new URL (e.g. a user dashboard after logging in), use a [redirect](#redirects) instead.
+:::
+
+Rewrites can be useful for showing the same content at multiple paths (e.g. `/products/shoes/men/` and `/products/men/shoes`/) without needing to maintain two different source files.
+
+Rewrites are also useful for SEO purposes and user experience. They allow you to display content that otherwise would require redirecting your visitor to a different page or would take them to a 404 page. One common use of rewrites is to show the same localized content for different variants of a language.
```

Comment:
```suggestion
Rewrites are also useful for SEO purposes and user experience. They allow you to display content that otherwise would require redirecting your visitor to a different page or would return a 404 status. One common use of rewrites is to show the same localized content for different variants of a language.
```